.github/workflows/ci.yml

```diff
@@ -12,4 +12,4 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - run: mvn clean install
+      - run: mvn clean install -B -V
```

Comment:
How about ` --strict-checksums` and others from https://github.com/trinodb/trino/blob/master/.github/workflows/ci.yml#L13?

Reply:
Those are good, but probably not necessary here, given it's a simpler project and built far less frequently. But feel free to send a PR if you think it's helpful. Would be nice to have a ready CI config for simple Maven projects.